gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/EncounterMapperTest.java

```diff
@@ -285,7 +285,7 @@ public void testMapValidEncounterWithNopatConfidentialityCodeWithinEhrCompositio
 
     @Test
     public void testMapValidEncounterWithNoscrubConfidentialityCodeWithinEhrCompositionExpectMetaSecurityNotAdded() {
-        final Meta stubbedMeta = MetaFactory.getMetaFor(META_WITHOUT_SECURITY, META_PROFILE);
+        final Meta stubbedMeta = MetaFactoryUtil.getMetaFor(META_WITHOUT_SECURITY, META_PROFILE);
```

Comment:
I think this should be:

```suggestion
        final Meta metaWithoutSecurity = MetaFactoryUtil.getMetaFor(META_WITHOUT_SECURITY, META_PROFILE);
```

Just for readability / maintainability 😄